### DISCUSSION TITLE
Piton Yılanları - Kaptan: tolga0102 (Tolga Arısalan)

### DISCUSSION THREAD

**Discussion by tolga0102** [sonsuzus/BilsemPython#63] [Genel]
Merhaba arkadaşlar,
Öncelikle istekli, kodlamada iyi olduğunu düşünen arkadaşlarımızı grubuma davet etmek istiyorum.
Ayrıca Salı günü bilişim dersinde olanların katılmasını tavsiye ederim. Şu anda Salı günü 18.00 - 19.20 derslerindeyim.
Katılmak isteyen diğerlerine de her zaman açığım onlar da ismini öyg grubunu yazsın. Azimli ve başarılı takım arkadaşları arıyorum.

<p align="center">
  <img width="350" height="350" src="https://user-images.githubusercontent.com/118377472/211187831-1d49c073-6a5f-43eb-9030-f6cba93d0d1d.png">
</p>

**Comment by Ersan0102** [comment]
Merhaba, ben Ersan. Piton Yılanları’na katılmak istiyorum😀👍

**Reply by tolga0102** [reply]
`Takıma hoş geldin Ersan. Umarım beraber iyi bir takım oluruz 😀`

**Comment by Atakanoyg1** [comment]
merhaba katılmak istiyorum adım Atakan ŞAHİN ÖYG 1 Çarşamba 3. ve 4. dersler

**Reply by tolga0102** [reply]
Takıma hoş geldin Atakan. Umarım beraber iyi bir takım oluruz.
Telefon numaranı yazarsan whatsapptan seni ekleyebilirim :)

**Reply by Atakanoyg1** [reply]
Teşekkürler Tolga, Telefon Numaram: 05305929010

**Comment by ghost** [comment]
Katılmak istiyorum
Ali İlker CAN 
ÖYG 1 Cumartesi 1 ve 2. derslere giriyorum.

**Reply by tolga0102** [reply]
Takıma hoş geldin Ali İlker. Umarım beraber iyi bir takım oluruz.
Telefon numaranı yazarsan whatsapptan seni ekleyebilirim :)

**Reply by ghost** [reply]
Kusura bakma, benim github hesabının şifresi kaybolmuştu yeni gördüm mesajı. 
05411483431

**Comment by efeozceylan** [comment]
Merhaba, takıma katılmak istiyorum;
Çarşamba 5 ve 6 oyg1 dersine giriyorum.

**Reply by tolga0102** [reply]
Takıma hoş geldin Efe. Umarım beraber iyi bir takım oluruz.
Telefon numaranı yazarsan whatsapptan seni ekleyebilirim :)
